constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/TransitionScope.kt

```diff
@@ -113,8 +117,24 @@ class TransitionScope internal constructor() {
         keyFramesObject.put("KeyCycles", keyCyclesArray)
     }
 
+    /**
+     *
```

Comment:
Do you plan to add doc comment? Same as the other variables below.

Reply:
Removed those empty blocks. Will add Javadoc in a follow-up PR, since there's a lot to comment.